VR Maze/Assets/Scripts/MazeClass.cs

```diff
@@ -0,0 +1,199 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Maze : MazeGenerate  {
+
+		List<mazeCell> mazeCellList = new List<mazeCell> ();
+		private int MazeSizeX;
+		private int MazeSizeZ;
+
+		//Constructor 0 arguments
+		public Maze(){
+			MazeSizeX = 0; //Sets maze size x
+			MazeSizeZ = 0; //Sets maze size z
+		}
+
+		//Constructor 2 arguments
+		public Maze(int mazeSizeX, int mazeSizeZ){
+			MazeSizeX =mazeSizeX; //Set maze size x 
+			MazeSizeZ =mazeSizeZ;
+		} 
+
+		public void addMazeCell(int x, int z, bool eastPath, bool southPath )
+		{
+			mazeCell currentCell = new mazeCell (x, z, southPath, eastPath);
+			mazeCellList.Add (currentCell);
+		}
+
+		public void generateMaze(){
```

Comment:
I'll eventually make a MazeGenerator class (or something to that effect) which will produce mazes, so this maze generation stuff will eventually be removed, just FYI